apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -1310,6 +1312,21 @@
       "NO_GROUPS": "Resource has no groups assigned",
       "ADD_GROUP": "Add group",
       "SELECT_RESOURCE": "Select resource"
+    },
+    "ADD_MEMBER_GROUP": {
+      "TITLE": "Add groups",
+      "ADD": "Add",
+      "CANCEL": "Cancel",
+      "SUCCESS": "Groups added"
+    },
+    "REMOVE_MEMBER_GROUP": {
+      "TITLE": "Remove groups",
+      "CANCEL": "Cancel",
+      "REMOVE": "Remove",
+      "DESCRIPTION": "Following groups will be removed.",
+      "ASK": "Do you want to proceed?",
+      "SUCCESS": "Groups removed"
```

Comment:
I do not have very good english, but maybe "Groups were removed" would sound better?